### DISCUSSION TITLE
Problem with nuxt v3.0.0-rc4

### DISCUSSION THREAD

**Discussion by amiria703** [nuxt/nuxt#16677] [Questions]
Hi!
I'm trying to make a webpage for myself using Nuxt 3.
I deployed it using Vercel and configured my domain with Cloudflare.
I have two problems with it:

1.The logo cannot be fetched:
```
GET https://amiria703.ir/_nuxt/logo.2e4e50f9.svg 404
```
2.Original app.vue renders but vue-router pages inside it doesn't:
```
Hydration completed but contains mismatches.
u @ entry-14c5207b.mjs:1
```
Live:
https://amiria703.ir
Source code:
https://github.com/amiria703/my-website

Any help is highly appreciated. 🙏❤️

**Comment by itsmnthn** [comment]
I see you wrote `assets/logo.svg` instead you should use `~/assets/logo.svg`

I think you should use `public` dir for static assets.

Also, for the hydration mismatch you should check it on dev mode and look for the element with classes and locate it in your code to see potential  cause and you should be able to fix it.

**Reply by amiria703** [reply]
Thanks for your quick answer! :pray::heart:
Issue with images are fixed and gone.
But still I have problems with hydration mismatch.
I don't understand what do you mean regarding this, Can you explain more?
In local, both `dev` and `build` builds work as expected.

**Reply by itsmnthn** [reply]
run project as `dev` and read the browser's console log to inspect where this hydration mismatch happens and look for possibilities in optional elements. Mostly this happens in `v-if`'s

**Comment by u007** [comment]
how did u deploy? im getting error with rc11

**Reply by TheAlexLichter** [reply]
Hydration errors can have manifold reasons. Feel free to check [this blogpost](https://blog.lichter.io/posts/vue-hydration-error/) (focus on Nuxt 2, but almost the same reasons).

**Reply by u007** [reply]
oh i got it working with pnpm i --shamefully-hoist
